pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -22,10 +22,11 @@ var systemRegistriesConfPath = builtinRegistriesConfPath
 // DO NOT change this, instead see systemRegistriesConfPath above.
 const builtinRegistriesConfPath = "/etc/containers/registries.conf"
 
-// Mirror represents a mirror. Mirrors can be used as pull-through caches for
-// registries.
-type Mirror struct {
-	// The mirror's URL.
+// Endpoint represents a registries terminal. Endpoints can be used as
+// pull-through caches for registries too, when specified as
+// `Registry.Endpoint`.
+type Endpoint struct {
```

Comment:
(Non-blocking: I’m not quite a fan of the `Endpoint` name, it is very non-specific; OTOH I don’t have any better suggestion right now, and it is not user-visible anyway.)

Reply:
The Moby PR uses the same terminology. For me, it makes things easier to read/translate.

Reply:
> The Moby PR uses the same terminology. For me, it makes things easier to read/translate.

Yeah; as long as it is used in the internal API and not in the user-visible configuration syntax, I can live with the name.